moPepGen/cli/call_variant_peptide.py

```diff
@@ -83,6 +83,11 @@ def add_subparser_call_variant(subparsers:argparse._SubParsersAction):
         action='store_true',
         help='For circRNA, only keep noncanonical peptides spaning the backsplicing site.'
     )
+    p.add_argument(
+        '--find-ass',
```

Comment:
I feel like ASS implies something different from novel ORFs, like it almost includes things that dont start with `M`. Is it better to try to keep the nomenclature the same? Like make this `include-coding-novel-orf` or something?

Reply:
Yeah very good point. I definitely saw the same phrase being used for both scenarios. Let me rename both to `--coding-novel-orf`.